hyperstart.go

```diff
@@ -430,6 +512,18 @@ func (h *hyper) startOneContainer(pod Pod, c Container) error {
 		return err
 	}
 
+	//TODO : Enter mount namespace
+
+	// Handle container mounts
+	fsmap, err := h.bindMountContainerMounts(pod.id, c.id, c.config.Mounts)
```

Comment:
We should pass `c.mounts` instead of `c.config.Mounts` because we don't want the initial config to be modified here.